wrapanapi/ec2.py

```diff
@@ -306,7 +306,7 @@ def is_vm_running(self, instance_id):
         try:
             running = self.vm_status(instance_id) in self.states['running']
             return running
-        except:
+        except: # noqa
```

Comment:
Any reason that we should not do 'except Exception' here?

Reply:
It was always like that but my linter complained so I added the #noqa. 
I will add Exception